docs/site/Model.md

```diff
@@ -190,9 +188,198 @@ class Product extends Entity {
+  <td>Boolean</td>
+  <td><code>true</code></td>
+  <td>
+    If true, prevents clients from setting the auto-generated ID value manually.
+  </td>
+  </tr>
+
+  <tr>
+  <td><code>http</code></td>
+  <td>String</td>
+  <td>None</td>
+  <td>Customized HTTP path for REST endpoints of this model.</td>
+  </tr>
+
+  <tr>
+    <td><code>strict</code></td>
+    <td>Boolean or String</td>
+    <td><code>false</code>.<br/>If the data source is backed by a relational database, then default is  <code>true</code>.</td>
+    <td>
+      Specifies whether the model accepts only predefined properties or not. One of:
+      <ul>
+      <li><code>true</code>: Only properties defined in the model are accepted. Use if you want to ensure the model accepts only predefined properties.
+      If you try to save a model instance with properties that are not predefined, LoopBack throws a ValidationError.
+      </li>
+      <li><code>false</code>: The model is an open model and accepts all properties, including ones not predefined in the model.
+        This mode is useful to store free-form JSON data to a schema-less database such as MongoDB.
+      </li>
+      <li><code>"filter"</code>: Only properties defined in the model are accepted.
+      If you load or save a model instance with properties that are not predefined, LoopBack will ignore them. This is particularly useful when dealing with old data that you wish to lose without a migration script.
+      </li>
+      </ul>
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>idInjection</code></td>
+    <td>Boolean</td>
+    <td><code>true</code></td>
+    <td>
+      Whether to automatically add an <code>id</code> property to the model:
+    <ul>
+    <li><code>true</code>: <code>id</code> property is added to the model automatically. This is the default.</li>
+    <li><code>false</code>: <code>id</code> property is not added to the model</li>
+    </ul>
+    See <a href="#id-properties">ID properties</a> for more information.  The <code>idInjection</code> property in <code>options</code> (if present) takes precedence.
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>name</code></td>
+    <td>String</td>
+    <td>None</td>
+    <td>Name of the model.</td>
+  </tr>
+
+  <tr>
+    <td><code>plural</code></td>
+    <td>String</td>
+    <td>Plural of <code>name</code> property using standard English conventions.</td>
+    <td>
+      Plural form of the model name.
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>properties</code></td>
+    <td>Object</td>
+    <td>N/A</td>
+    <td>
+      Even though properties can be defined in model decorator, we don't recommend this way since it's inconsistent. <b>Please do define <code>properties</code> by <code>@property Decorator</code></b>.
+    </td>
+  </tr>
+
+  <tr>
+    <td><code>scopes</code></td>
+    <td>Object</td>
+    <td>N/A</td>
+    <td>See <a href="https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#scopes">Scopes</a> in lb3 docs.</td>
```

Comment:
I think this property should be move the "Not Available" section, since it's not supported by LB4 (yet).

WDYT?

Reply:
I agree.  But I think it kinda 'works' in model decorator as I tested it out before. I don't know if that's because we handle `properties`[here](https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/model-builder.js#L126). It would still get overwritten by property decorators anyways. So it makes more sense for me to remove it from the available list.